README.rst

```diff
@@ -3,24 +3,25 @@
 
 Raiden Scenario Player
 ======================
-The Raiden Scneario Player is an integration testing tool written in Python 3. It allows testing of various test scenarios, and is
+The Raiden Scenario Player is an integration testing tool written in Python 3. It allows testing of various test scenarios, and is
 an integral component in the raiden test suite.
```

Comment:
is it "of the raiden test suite"? Can you spell Raiden with an capital "R"

Reply:
I can and I shall! and `of` is correct, yes.